app/models/call_for_evidence_response.rb

```diff
@@ -0,0 +1,53 @@
+class CallForEvidenceResponse < ApplicationRecord
```

Comment:
I think this model should not be part of this PR - to be added in the outcome PR